autopts/pybtp/btp/btp.py

```diff
@@ -586,6 +618,12 @@ def event_handler(hdr, data):
             cb(stack.gatt_cl, data[0], hdr.data_len)
             return True
 
+    elif hdr.svc_id == defs.BTP_SERVICE_ID_IAS:
+        if hdr.op in IAS_EV and stack.ias:
+            cb = IAS_EV[hdr.op]
+            cb(stack.ias, data[0], hdr.data_len)
+            return True
+
```

Comment:
What about handling events from other services you add (VCS, VOCS, AICS)?

Reply:
Done